src/webgpu/api/operation/buffers/map_oom.spec.ts

```diff
@@ -125,15 +126,19 @@ an out-of-memory error if allocation fails.
       .combine('usage', kBufferUsages)
   )
   .fn(async t => {
-    const { usage, size } = t.params;
+    const { oom, usage, size } = t.params;
 
-    const buffer = t.expectGPUError('out-of-memory', () =>
-      t.device.createBuffer({ mappedAtCreation: true, size, usage })
+    const buffer = t.expectGPUError(
+      'out-of-memory',
+      () => t.device.createBuffer({ mappedAtCreation: true, size, usage }),
+      oom
     );
 
-    // Smaller range inside a too-big mapping
     const mapping = buffer.getMappedRange(0, 16);
     t.expect(mapping.byteLength === 16);
-    buffer.unmap();
-    t.expect(mapping.byteLength === 0, 'Mapping should be detached');
+
+    t.expectGPUError('validation', () => buffer.unmap(), oom);
+    if (mapping !== undefined) {
+      t.expect(mapping.byteLength === 0, 'Mapping should be detached');
+    }
```

Comment:
```suggestion
    t.expect(mapping.byteLength === 0, 'Mapping should be detached');
```
No longer needed I think